Tools/cases_generator/parser.py

```diff
@@ -68,10 +68,25 @@ class Block(Node):
     pass
 
 
+@dataclass
+class StackVarTypeLiteral(Node):
+    literal: str
+
+
+@dataclass
+class StackVarTypeIndex(Node):
+    array: Literal["locals"] | Literal["consts"]
+    index: str
+
+
+StackVarType = StackVarTypeLiteral | StackVarTypeIndex
```

Comment:
Not required but good to help the type checker.
```suggestion
StackVarType: TypeAlias = StackVarTypeLiteral | StackVarTypeIndex
```